src/utils/workspacUtils.ts

```diff
@@ -4,9 +4,16 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as vscode from 'vscode';
+import { ext } from '../extensionVariables';
+import { nonNullValue } from './nonNull';
 
 // tslint:disable-next-line: export-name
 export function getRootPath(): string | undefined {
     // if this is a multi-root workspace, return undefined
     return vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length === 1 ? vscode.workspace.workspaceFolders[0].uri.fsPath : undefined;
 }
+
+export function getBatchSizeSetting(): number {
```

Comment:
I would have put this in constants.ts but languageService.ts can't take a dependency on vscode (languageService.ts consumes something from constants.ts)